src/components/FullScreenGallery.js

```diff
@@ -29,7 +29,8 @@ export const FullScreenGallery = ({
   onPrevious = () => {},
   initialSlide = 1,
   backgroundColor,
-  portalId
+  portalId,
+  isDesignerAppointed
```

Comment:
Similar to the other PR, we might want to pass a classes prop from the parent app to override styles here. We can huddle about it.